about.html

```diff
@@ -1,64 +1,109 @@
+      <header>
+        <div class="navigation">
+          <img src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/lambda-black.png?raw=true" alt="This is the Lambda logo.">
+            <nav>
+              <a href="index.html">Home</a>
+              <a href="#">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+            </nav>
+        </div>
+        <div class="image">
+          <img src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/jumbo-about.png?raw=true" alt="This is our header image!">
+        </div>
+      </header>
+      <section class="top-content-about">
+        <div class="text-container-about">
+          <h2>About</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </div>
+      </section>
+      <section class="middle-content-about">
+        <div class="about-box">
+          <img src="img/about-plan.png" alt="strategy">
+    
+          <h2>Strategy</h2>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
+      
+        </div>
+        <div class="about-box">
+          <img src="img/about-working.png" alt="strategy">
+        
+          <h2>How We Work</h2>
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+          <button>Learn More</button>
         
+        </div>
+        <div class="about-box">
+          <img src="img/about-office.png" alt="strategy">
+          
+            <h2>Places We Work</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+          
+        </div>
+        <div class="about-box">
+          <img src="img/about-meeting.png" alt="strategy">
+            
+            <h2>Collaboration</h2>
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+            <button>Learn More</button>
+        </div>
+      </section>
+      <section class="bottom-content-about">
+        <h2>Let's Work Together</h2>
+
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </section>  
       <footer>
```

Comment:
I think your class naming is really good. It makes things really clear for future programmers!